config/opal_save_version.m4

```diff
@@ -55,7 +55,7 @@ AC_DEFUN([OPAL_SAVE_VERSION], [
         [The repository version ]$2)
     AC_DEFINE_UNQUOTED($1[_TARBALL_VERSION], ["$]$1[_TARBALL_VERSION"],
         [Tarball filename version string of ]$2)
-    AC_DEFINE_UNQUOTED($1[_VERSION], ["$]$1[_RELEASE_VERSION"],
+    AC_DEFINE_UNQUOTED($1[_VERSION], ["$]$1[_VERSION"],
```

Comment:
This sets off alarm bells that maybe there was a reason we used RELEASE, something around Autoconf stupid, but I can't remember.  @jsquyres, do you?

Reply:
It actually looks like a copy-n-paste error from line 50, above (from the MAJOR/MINOR/RELEASE version numbers, above).